libarchive/archive_write_set_options.3

```diff
@@ -698,6 +698,12 @@ It should not be used except for testing purposes.
 .It Cm hdrcharset
 The value is used as a character set name that will be
 used when translating file names.
+.It Cm high-resolution-time
+This boolean option enables or disables writing high-resolution
+timestamps that may not be compatible with other Zip implementations.
+.Pp
+To note is that high-resolution timestamps are necessary to support
+dates beyond 2038 so in that case they're always written.
```

Comment:
We should not require an option to support dates beyond 2038.

Reply:
Looking at your implementation, you're not requiring this to be set for dates beyond 2038.  This controls the writing of timestamps with accuracy finer than 1s.

The "compatible" comment is not relevant.  Zip is pretty well designed in this regard; readers just ignore extensions they don't understand.  Compatibility is only an issue when it might cause a reader to fail to read the entry at all.